src/AppBundle/Entity/OrderRepository.php

```diff
@@ -8,26 +8,27 @@
 
 class OrderRepository extends EntityRepository
 {
-    public function getWaitingOrdersForRestaurant(Restaurant $restaurant, \DateTime $date = null)
+    public function getWaitingOrdersForRestaurant(Restaurant $restaurant, \DateTime $now = null)
     {
+        if (null === $now) {
+            $now = new \DateTime();
+        }
+
         $qb = $this->createQueryBuilder('o');
         $qb
             ->join(Delivery::class, 'd', Expr\Join::WITH, 'd.order = o.id')
             ->andWhere('o.restaurant = :restaurant')
             ->andWhere($qb->expr()->in('o.status', [
                 Order::STATUS_WAITING,
-                Order::STATUS_ACCEPTED
+                Order::STATUS_ACCEPTED,
+                Order::STATUS_REFUSED
```

Comment:
I think we should show `STATUS_READY` too, because the restaurant owner still need it when the courier arrives to print the receipt.

I think we will show the ready orders that don't have been picked (`STATUS_PICKED` on delivery). Maybe we could only filter on delivery status? All items that are `STATUS_WAITING` or `STATUS_DISPATCHED` are in charge of the restaurant.